docs/source/conf.py

```diff
@@ -146,6 +146,10 @@
 # If true, "(C) Copyright ..." is shown in the HTML footer. Default is True.
 html_show_copyright = False
 
+# Add the last updated information to the bottom of pages. Blank string implies
+# '%b %d, %Y' format.
+html_last_updated_fmt = ""
+
```

Comment:
What is %b %d %Y format?
Note that Month, Day, Year format is not internationally acceptable. Instead the ISO YYYY-MM-DD format could be used.

Reply:
:+1: